scripts/test/lint.py

```diff
@@ -253,12 +253,27 @@ def check(self, path):
         return True
 
 
+class PackageIdNotMatchingFolderOrNuspecName(Lint):
+    name = "package ID doesn't match package folder name or nuspec file name"
+    recommendation = "make sure the package ID is the same as the package folder name and the nuspec file name"
+
+    def check(self, path):
+
```

Comment:
Minor nitpick. Could we remove the extra space between `check` and `dom` as well as `folder` and `return`?

Reply:
Updated